### PR TITLE
More deployment examples of Minio Tenant with AutoCert custom Certs

### PR DESCRIPTION
**Pull request by Alevsk**

### CODE REVIEW

**Review by nitisht — COMMENTED** [review]
@Alevsk can we add a file `examples.md` under the `docs` directory with a high level write up on what each of these examples do?

**Comment by Alevsk** [comment]
> @Alevsk can we add a file `examples.md` under the `docs` directory with a high level write up on what each of these examples do?

Sure, I'll do

**Review by nitisht — DISMISSED** [review]
@Alevsk looks good, added some small changes in your fork https://github.com/Alevsk/minio-operator/pull/1

**Comment by Alevsk** [comment]
> @Alevsk looks good, added some small changes in your fork [Alevsk#1](https://github.com/Alevsk/minio-operator/pull/1)

done

File: examples/tenant-with-autocert-encryption-enabled.yaml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    consoleSecret:
+      name: minio-autocert-encryption-console-secret
+  ## Define configuration for KES (stateless and distributed key-management system)
+  ## Refer https://github.com/minio/kes
+  kes:
+    image: minio/kes:v0.12.0
+    replicas: 2
+    kesSecret:
+      name: minio-autocert-encryption-kes-config
+    metadata:
```
Comment (nitisht) [inline examples/tenant-with-autocert-encryption-enabled.yaml:175]: KES metadata to be removed from here as well.

**Review by nitisht — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**